sdk/storage/Azure.Storage.Files.Shares/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Naming", "AZC0034:Type name conflicts", Justification = "Storage-specific enum that shares name with Blobs storage", Scope = "type", Target = "~T:Azure.Storage.Files.Shares.Models.CopyStatus")]
```

Comment:
check which of these exclusions are still needed given the new analyzer package.